tilt-provider.json

```diff
@@ -1,18 +1,38 @@
-{
-  "name": "vsphere",
-  "config": {
-    "version": "v1.10.99",
-    "image": "gcr.io/k8s-staging-capi-vsphere/cluster-api-vsphere-controller",
-    "live_reload_deps": [
-      "main.go",
-      "go.mod",
-      "go.sum",
-      "apis",
-      "controllers",
-      "feature",
-      "internal",
-      "pkg"
-    ],
-    "label": "CAPV"
+[
+  {
+    "name": "vsphere",
+    "config": {
+      "version": "v1.10.99",
+      "image": "gcr.io/k8s-staging-capi-vsphere/cluster-api-vsphere-controller",
+      "live_reload_deps": [
+        "main.go",
+        "go.mod",
+        "go.sum",
+        "apis",
+        "controllers",
+        "feature",
+        "internal",
+        "pkg"
+      ],
+      "label": "CAPV"
+    }
+  },
+  {
+    "name": "vsphere-supervisor",
+    "config": {
+      "version": "v1.10.99",
+      "image": "gcr.io/k8s-staging-capi-vsphere/cluster-api-vsphere-controller",
+      "live_reload_deps": [
+        "main.go",
+        "go.mod",
+        "go.sum",
+        "api",
+        "contrib",
+        "controllers",
+        "pkg"
+      ],
+      "label": "CAPV",
```

Comment:
We might want a different label that doesn't conflict. But not sure

Reply:
Maybe this also blocks deploying both at the same time, which would be probably good

Reply:
It is not possible to deploy both at the same time